extensions/amp-date-display/1.0/README.md

```diff
@@ -0,0 +1,270 @@
+---
+$category@: presentation
+formats:
+  - websites
+teaser:
+  text: The Bento Date Display component displays time data that you can render in your page.
+experiental: true
+bento: true
```

Comment:
these two annotations should not be needed

Reply:
updated